### ISSUE TITLE
Format Text in Non-Selected Chiclets

### ISSUE THREAD

**Issue by Blak9** [microsoft/PowerBI-visuals-ChicletSlicer#102]
Great visual, however, I would like to propose the following enhancement:

* To be able to format text color for each of the Chiclet coloroptions. 

F.i. if I format my selected chiclet black and the text white (or any other light color), I can't leave the non-selected chiclet white, because the text would be unreadable. I would apppreciate if I could format that to black (or any other color).

Basically: for each of the Chiclet coloroptions (Unselected and Disabled) I'd like coloroptions for text and hover. 

Many thanks in advance...
